<html>
<head>
<title>TestNG:  Test</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>Test</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>10/6/0</td>
</tr><tr>
<td>Started on:</td><td>Wed Jul 28 13:27:25 IST 2021</td>
</tr>
<tr><td>Total time:</td><td>764 seconds (764599 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='com.saucedemo.testcases.TC_NegativeCase_EmptyCart.EmptyCart()'><b>EmptyCart</b><br>Test class: com.saucedemo.testcases.TC_NegativeCase_EmptyCart<br>Test method: Verify whether the user is able proceed order without adding products into cart</td>
<td><div><pre>java.lang.AssertionError: Cart is Empty expected [true] but found [false]
	at com.saucedemo.pageObjects.Saucedemo_Cart.click_checkOut(Saucedemo_Cart.java:25)
	at com.saucedemo.testcases.TC_NegativeCase_EmptyCart.EmptyCart(TC_NegativeCase_EmptyCart.java:58)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 30 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1699449247", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1699449247'><pre>java.lang.AssertionError: Cart is Empty expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:99)
	at org.testng.Assert.failNotEquals(Assert.java:1037)
	at org.testng.Assert.assertTrue(Assert.java:45)
	at com.saucedemo.pageObjects.Saucedemo_Cart.click_checkOut(Saucedemo_Cart.java:25)
	at com.saucedemo.testcases.TC_NegativeCase_EmptyCart.EmptyCart(TC_NegativeCase_EmptyCart.java:58)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
	at org.testng.TestRunner.privateRun(TestRunner.java:764)
	at org.testng.TestRunner.run(TestRunner.java:585)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
	at org.testng.SuiteRunner.run(SuiteRunner.java:286)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.runSuites(TestNG.java:1069)
	at org.testng.TestNG.run(TestNG.java:1037)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>73</td>
<td>com.saucedemo.testcases.TC_NegativeCase_EmptyCart@3aefe5e5</td></tr>
<tr>
<td title='com.saucedemo.testcases.TC_ProblemUser_AddToCart.Login_ProblemUser()'><b>Login_ProblemUser</b><br>Test class: com.saucedemo.testcases.TC_ProblemUser_AddToCart<br>Test method: Verify whether the Problem user is able add products into cart</td>
<td><div><pre>java.lang.AssertionError: driver.findElement(text_error).getText() expected [true] but found [false]
	at com.saucedemo.pageObjects.Saucedemo_CheckOut.verify_ErrorMsg(Saucedemo_CheckOut.java:58)
	at com.saucedemo.testcases.TC_ProblemUser_AddToCart.Login_ProblemUser(TC_ProblemUser_AddToCart.java:67)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 30 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1211363315", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1211363315'><pre>java.lang.AssertionError: driver.findElement(text_error).getText() expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:99)
	at org.testng.Assert.failNotEquals(Assert.java:1037)
	at org.testng.Assert.assertTrue(Assert.java:45)
	at com.saucedemo.pageObjects.Saucedemo_CheckOut.verify_ErrorMsg(Saucedemo_CheckOut.java:58)
	at com.saucedemo.testcases.TC_ProblemUser_AddToCart.Login_ProblemUser(TC_ProblemUser_AddToCart.java:67)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
	at org.testng.TestRunner.privateRun(TestRunner.java:764)
	at org.testng.TestRunner.run(TestRunner.java:585)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
	at org.testng.SuiteRunner.run(SuiteRunner.java:286)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.runSuites(TestNG.java:1069)
	at org.testng.TestNG.run(TestNG.java:1037)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>37</td>
<td>com.saucedemo.testcases.TC_ProblemUser_AddToCart@4690b489</td></tr>
<tr>
<td title='com.saucedemo.testcases.TC_NegativeCase_Login.Login_StandardUser()'><b>Login_StandardUser</b><br>Test class: com.saucedemo.testcases.TC_NegativeCase_Login<br>Test method: Verify whether the Fields accepting invalid values</td>
<td><div><pre>java.lang.AssertionError: Epic sadface: Username and password do not match any user in this service expected [true] but found [false]
	at com.saucedemo.pageObjects.Saucedemo_Login.verify_ErrorMsg(Saucedemo_Login.java:61)
	at com.saucedemo.testcases.TC_NegativeCase_Login.Login_StandardUser(TC_NegativeCase_Login.java:23)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 30 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace824348035", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace824348035'><pre>java.lang.AssertionError: Epic sadface: Username and password do not match any user in this service expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:99)
	at org.testng.Assert.failNotEquals(Assert.java:1037)
	at org.testng.Assert.assertTrue(Assert.java:45)
	at com.saucedemo.pageObjects.Saucedemo_Login.verify_ErrorMsg(Saucedemo_Login.java:61)
	at com.saucedemo.testcases.TC_NegativeCase_Login.Login_StandardUser(TC_NegativeCase_Login.java:23)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
	at org.testng.TestRunner.privateRun(TestRunner.java:764)
	at org.testng.TestRunner.run(TestRunner.java:585)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
	at org.testng.SuiteRunner.run(SuiteRunner.java:286)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.runSuites(TestNG.java:1069)
	at org.testng.TestNG.run(TestNG.java:1037)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>2</td>
<td>com.saucedemo.testcases.TC_NegativeCase_Login@47d90b9e</td></tr>
<tr>
<td title='com.saucedemo.testcases.TC_ProblemUser_AddToCartMultiple.ProblemUser()'><b>ProblemUser</b><br>Test class: com.saucedemo.testcases.TC_ProblemUser_AddToCartMultiple<br>Test method: Verify whether the Problem  is able add multiple products into cart</td>
<td><div><pre>java.lang.AssertionError: Cart count is not matching expected [true] but found [false]
	at com.saucedemo.pageObjects.Saucedemo_HomePage.verify_cartCount(Saucedemo_HomePage.java:238)
	at com.saucedemo.testcases.TC_ProblemUser_AddToCartMultiple.ProblemUser(TC_ProblemUser_AddToCartMultiple.java:52)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 30 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1589745212", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1589745212'><pre>java.lang.AssertionError: Cart count is not matching expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:99)
	at org.testng.Assert.failNotEquals(Assert.java:1037)
	at org.testng.Assert.assertTrue(Assert.java:45)
	at com.saucedemo.pageObjects.Saucedemo_HomePage.verify_cartCount(Saucedemo_HomePage.java:238)
	at com.saucedemo.testcases.TC_ProblemUser_AddToCartMultiple.ProblemUser(TC_ProblemUser_AddToCartMultiple.java:52)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
	at org.testng.TestRunner.privateRun(TestRunner.java:764)
	at org.testng.TestRunner.run(TestRunner.java:585)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
	at org.testng.SuiteRunner.run(SuiteRunner.java:286)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.runSuites(TestNG.java:1069)
	at org.testng.TestNG.run(TestNG.java:1037)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>33</td>
<td>com.saucedemo.testcases.TC_ProblemUser_AddToCartMultiple@79b06cab</td></tr>
<tr>
<td title='com.saucedemo.testcases.TC_ProblemUser_RemovingAndAddingInCart.Problemuser_Removing()'><b>Problemuser_Removing</b><br>Test class: com.saucedemo.testcases.TC_ProblemUser_RemovingAndAddingInCart<br>Test method: Verify whether the Problem user is able add multiple products into cart and remove</td>
<td><div><pre>java.lang.AssertionError: Not Matching expected [true] but found [false]
	at com.saucedemo.pageObjects.Saucedemo_Cart.verify_SelectedProductsAreShowing(Saucedemo_Cart.java:55)
	at com.saucedemo.testcases.TC_ProblemUser_RemovingAndAddingInCart.Problemuser_Removing(TC_ProblemUser_RemovingAndAddingInCart.java:57)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 30 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace854977732", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace854977732'><pre>java.lang.AssertionError: Not Matching expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:99)
	at org.testng.Assert.failNotEquals(Assert.java:1037)
	at org.testng.Assert.assertTrue(Assert.java:45)
	at com.saucedemo.pageObjects.Saucedemo_Cart.verify_SelectedProductsAreShowing(Saucedemo_Cart.java:55)
	at com.saucedemo.testcases.TC_ProblemUser_RemovingAndAddingInCart.Problemuser_Removing(TC_ProblemUser_RemovingAndAddingInCart.java:57)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
	at org.testng.TestRunner.privateRun(TestRunner.java:764)
	at org.testng.TestRunner.run(TestRunner.java:585)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
	at org.testng.SuiteRunner.run(SuiteRunner.java:286)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.runSuites(TestNG.java:1069)
	at org.testng.TestNG.run(TestNG.java:1037)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>34</td>
<td>com.saucedemo.testcases.TC_ProblemUser_RemovingAndAddingInCart@3eb7fc54</td></tr>
<tr>
<td title='com.saucedemo.testcases.TC_NegativeCase_CheckOut.StandardUser_CheckOut_NC()'><b>StandardUser_CheckOut_NC</b><br>Test class: com.saucedemo.testcases.TC_NegativeCase_CheckOut<br>Test method: Verify whether the checkout page is accepting invalid/empty values.</td>
<td><div><pre>java.lang.AssertionError: Error Message is not Showing expected [true] but found [false]
	at com.saucedemo.pageObjects.Saucedemo_CheckOut.verify_CheckOutErrorMsg(Saucedemo_CheckOut.java:74)
	at com.saucedemo.testcases.TC_NegativeCase_CheckOut.StandardUser_CheckOut_NC(TC_NegativeCase_CheckOut.java:127)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 30 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace2106535660", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace2106535660'><pre>java.lang.AssertionError: Error Message is not Showing expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:99)
	at org.testng.Assert.failNotEquals(Assert.java:1037)
	at org.testng.Assert.assertTrue(Assert.java:45)
	at com.saucedemo.pageObjects.Saucedemo_CheckOut.verify_CheckOutErrorMsg(Saucedemo_CheckOut.java:74)
	at com.saucedemo.testcases.TC_NegativeCase_CheckOut.StandardUser_CheckOut_NC(TC_NegativeCase_CheckOut.java:127)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
	at org.testng.TestRunner.privateRun(TestRunner.java:764)
	at org.testng.TestRunner.run(TestRunner.java:585)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
	at org.testng.SuiteRunner.run(SuiteRunner.java:286)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.runSuites(TestNG.java:1069)
	at org.testng.TestNG.run(TestNG.java:1037)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>64</td>
<td>com.saucedemo.testcases.TC_NegativeCase_CheckOut@1184ab05</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-passed'>
<tr><td colspan='4' align='center'><b>PASSED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='com.saucedemo.testcases.TC_Entitlement_LockedOutUser.Login_LockedOutUser()'><b>Login_LockedOutUser</b><br>Test class: com.saucedemo.testcases.TC_Entitlement_LockedOutUser<br>Test method: Verify whether the Locked Out user is able not to login</td>
<td></td>
<td>3</td>
<td>com.saucedemo.testcases.TC_Entitlement_LockedOutUser@4ddced80</td></tr>
<tr>
<td title='com.saucedemo.testcases.TC_PerformanceGlitchUser_AddToCart.Login_PerformanceGlitchUser()'><b>Login_PerformanceGlitchUser</b><br>Test class: com.saucedemo.testcases.TC_PerformanceGlitchUser_AddToCart<br>Test method: Verify whether the Performance Glitch user is able add products into cart</td>
<td></td>
<td>58</td>
<td>com.saucedemo.testcases.TC_PerformanceGlitchUser_AddToCart@7f552bd3</td></tr>
<tr>
<td title='com.saucedemo.testcases.TC_Entitlement_StandardUser.Login_StandardUser()'><b>Login_StandardUser</b><br>Test class: com.saucedemo.testcases.TC_Entitlement_StandardUser<br>Test method: Verify whether the Standard user is able to login successfuly</td>
<td></td>
<td>34</td>
<td>com.saucedemo.testcases.TC_Entitlement_StandardUser@65466a6a</td></tr>
<tr>
<td title='com.saucedemo.testcases.TC_Entitlement_PerformanceGlitchUser.Login_StandardUser()'><b>Login_StandardUser</b><br>Test class: com.saucedemo.testcases.TC_Entitlement_PerformanceGlitchUser<br>Test method: Verify whether the Performance Glitch user is able to login successfuly</td>
<td></td>
<td>41</td>
<td>com.saucedemo.testcases.TC_Entitlement_PerformanceGlitchUser@1534f01b</td></tr>
<tr>
<td title='com.saucedemo.testcases.TC_Entitlement_ProblemUser.Login_StandardUser()'><b>Login_StandardUser</b><br>Test class: com.saucedemo.testcases.TC_Entitlement_ProblemUser<br>Test method: Verify whether the Problem user is able to login successfuly</td>
<td></td>
<td>34</td>
<td>com.saucedemo.testcases.TC_Entitlement_ProblemUser@78e117e3</td></tr>
<tr>
<td title='com.saucedemo.testcases.TC_StandardUser_AddToCart.Login_StandardUser()'><b>Login_StandardUser</b><br>Test class: com.saucedemo.testcases.TC_StandardUser_AddToCart<br>Test method: Verify whether the Standard user is able add products into cart</td>
<td></td>
<td>48</td>
<td>com.saucedemo.testcases.TC_StandardUser_AddToCart@2ea227af</td></tr>
<tr>
<td title='com.saucedemo.testcases.TC_PerformanceGlitchUser_AddToCartMultiple.PerformaceGlitchUser()'><b>PerformaceGlitchUser</b><br>Test class: com.saucedemo.testcases.TC_PerformanceGlitchUser_AddToCartMultiple<br>Test method: Verify whether the Performance Glitch user is able add products into cart</td>
<td></td>
<td>59</td>
<td>com.saucedemo.testcases.TC_PerformanceGlitchUser_AddToCartMultiple@3c22fc4c</td></tr>
<tr>
<td title='com.saucedemo.testcases.TC_PerformanceGlitchUser_RemovingAndAddingInCart.PerformanceGlitchUser_Removing()'><b>PerformanceGlitchUser_Removing</b><br>Test class: com.saucedemo.testcases.TC_PerformanceGlitchUser_RemovingAndAddingInCart<br>Test method: Verify whether the PerformanceGlitch user is able add multiple products into cart and remove</td>
<td></td>
<td>60</td>
<td>com.saucedemo.testcases.TC_PerformanceGlitchUser_RemovingAndAddingInCart@460d0a57</td></tr>
<tr>
<td title='com.saucedemo.testcases.TC_StandadUser_AddToCartMultiple.StandardUser_Multiple()'><b>StandardUser_Multiple</b><br>Test class: com.saucedemo.testcases.TC_StandadUser_AddToCartMultiple<br>Test method: Verify whether the Standard user is able add multiple products into cart</td>
<td></td>
<td>50</td>
<td>com.saucedemo.testcases.TC_StandadUser_AddToCartMultiple@4386f16</td></tr>
<tr>
<td title='com.saucedemo.testcases.TC_StandardUser_RemovingAndAddingInCart.StandardUser_Removing()'><b>StandardUser_Removing</b><br>Test class: com.saucedemo.testcases.TC_StandardUser_RemovingAndAddingInCart<br>Test method: Verify whether the Standard user is able add multiple products into cart and remove</td>
<td></td>
<td>49</td>
<td>com.saucedemo.testcases.TC_StandardUser_RemovingAndAddingInCart@363ee3a2</td></tr>
</table><p>
</body>
</html>